docs/vc-http-api.yml

```diff
@@ -113,6 +113,65 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /presentations/available:
+    post:
+      tags:
+        - Holder
+      summary: Notifies a holder of an available presentation.
+      operationId: notifyPresentationAvailable
+      description:
+        This API is the first part of a holder to holder credential exchange over http.
+        The server MAY reply with a query for the client to proceed with a presentation.
+      requestBody:
+        description: Details the client provides to the server to help it decide if the presentation should be made.
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/NotifyPresentationAvailableRequest"
+
+      responses:
+        "200":
+          description: Proceed with presentation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/NotifyPresentationAvailableResponse"
+        "400":
```

Comment:
Do we need a 401 Unauthorized (I don't know who you are, caller) and a 404 (I don't know what this presentation you are trying to send me is, caller)?

Reply:
Probably we need a lot fo 400 level status codes... prefer to tackle them in a separate PR, please open an issue to discuss further.